docs/manual/other-topics/scopes.md

```diff
@@ -166,10 +166,25 @@ YourMode.addScope('scope2', {
 Using `.scope('scope1', 'scope2')` will yield the following WHERE clause:
 
 ```sql
-WHERE firstName = 'bob' AND age > 30 LIMIT 10
+WHERE firstName = 'bob' AND age < 30 LIMIT 10
 ```
 
-Note how `limit` and `age` are overwritten by `scope2`, while `firstName` is preserved. The `limit`, `offset`, `order`, `paranoid`, `lock` and `raw` fields are overwritten, while `where` is shallowly merged (meaning that identical keys will be overwritten). The merge strategy for `include` will be discussed later on.
+Note how `limit` and `age` are overwritten by `scope2`, while `firstName` is preserved. The `limit`, `offset`, `order`, `paranoid`, `lock` and `raw` fields are overwritten, while `where` is by default shallowly merged (meaning that identical keys will be overwritten). If the flag `whereMergeStrategy` is set to `and` (on the model or on the sequelize instance), `where` fields will be merged using the `and` operator.
+
+For instance, if `YourModel` was initialized as such:
+
+```js
+YourModel.init({ /* attributes */ }, {
+  // ... other init options
+  whereMergeStrategy: 'and',
+  })
```

Comment:
Last nit, otherwise this is good to go for me

```suggestion
});
```

Reply:
Done